cli/azd/cmd/provision.go

```diff
@@ -31,6 +33,12 @@ func (i *provisionFlags) Bind(local *pflag.FlagSet, global *internal.GlobalComma
 }
 
 func (i *provisionFlags) bindNonCommon(local *pflag.FlagSet, global *internal.GlobalCommandOptions) {
+	local.StringToStringVarP(
+		&i.parameters,
+		"parameters",
+		"p",
+		map[string]string{},
+		"Template parameters as `parameter=value` pairs.")
```

Comment:
What are you thoughts on using this single `--parameters` param vs using a string array param that would allow users to set multiple `--parameter` flags?

What is the syntax and delimiter for multiple values in single param mode? comma? semi-colon?

**Single**
azd provision --parameters A=1;B=2

**Multiple**
azd provision --parameter A=1 --parameter --B=2

Seems like many other dev tools like docker and terraform itself prefer the latter implementation.

Reply:
I think we also use the second form in cases where we expect multiple arguments elsewhere in the tool.  Like @wbreza I prefer this and think it is closer to what both the AZ CLI and Terraform do.

Reply:
Spf13/pflags' `StringToString(Var(P))` methods do allow you to pass multiple `--parameters` flags. IT's what the GitHub CLI and others use: https://go.dev/play/p/DbctVd8Lxpl